libraries/AP_BoardConfig/px4_drivers.cpp

```diff
@@ -518,7 +537,7 @@ void AP_BoardConfig::px4_sensor_error(const char *reason)
     */
     while (true) {
         printf("Sensor failure: %s\n", reason);
-        hal.console->printf("Sensor failure: %s\n", reason);
+        hal.console->printf("Check BRD_TYPE: %s\n", reason);
```

Comment:
I don't like this change. What if BRD_TYPE is correct and it is indeed a sensor failure? I know it will still display the "No X found" message but it will seem like the problem is BRD_TYPE.
I think that the previous message with an additional "Check if BRD_TYPE is correct" would be much better.